runtime/tutorials/how_to_with_npm/react.md

```diff
@@ -65,180 +89,205 @@ router
     context.response.body = data;
   })
   .get("/api/:dinosaur", (context) => {
-    if (context?.params?.dinosaur) {
-      const found = data.find((item) =>
-        item.name.toLowerCase() === context.params.dinosaur.toLowerCase()
-      );
-      if (found) {
-        context.response.body = found;
-      } else {
-        context.response.body = "No dinosaurs found.";
-      }
+    if (!context?.params?.dinosaur) {
+      context.response.body = "No dinosaur name provided.";
     }
+
+    const dinosaur = data.find((item) =>
+      item.name.toLowerCase() === context.params.dinosaur.toLowerCase()
+    );
+
+    context.response.body = dinosaur ? dinosaur : "No dinosaur found.";
   });
+```
 
+Finally, at the bottom of the same file, create a new `Application` instance and
+attach the routes we just defined to the application using
+`app.use(router.routes())` and start the server listening on port 8000:
+
+```ts title="main.ts"
 const app = new Application();
-app.use(oakCors()); // Enable CORS for All Routes
+app.use(oakCors());
 app.use(router.routes());
 app.use(router.allowedMethods());
 
 await app.listen({ port: 8000 });
 ```
 
-This is a very simple API server using [`oak`](https://deno.land/x/oak) that
-will return dinosaur information based on the route. Let's start the API server:
+You can run the API server with `deno run --allow-env --allow-net api/main.ts`.
+We'll create a task to run this command in the background and update the dev
+task to run both the React app and the API server.
 
-```shell
-deno run --allow-env --allow-net api/main.ts
-```
+In your `deno.json` file, update the `tasks` field to include the following:
 
-If we go to `localhost:8000`, we see:
+```json
+{
+  "tasks": {
+    "dev": "deno task dev:api & deno task dev:vite",
+    "dev:api": "deno run --allow-env --allow-net api/main.ts",
+    "dev:vite": "deno run -A npm:vite",
+    ...
+```
```

Comment:
If we define it as jsonc then we can use comments

````suggestion
```jsonc
{
  "tasks": {
    "dev": "deno task dev:api & deno task dev:vite",
    "dev:api": "deno run --allow-env --allow-net api/main.ts",
    "dev:vite": "deno run -A npm:vite",
    // ...
  }
}
```
````